scripts/coverage.sh

```diff
@@ -35,10 +35,13 @@ if [[ -d target/cov ]]; then
 fi
 rm -rf target/cov/$reportName
 
-source ci/rust-version.sh nightly
+if [ -z $SOLANA_RUST_VERSION ]; then
+  source ci/rust-version.sh nightly
+  SOLANA_RUST_VERSION="+$rust_nightly"
+fi
 
-RUST_LOG=solana=trace _ cargo +$rust_nightly test --target-dir target/cov --no-run "${packages[@]}"
-RUST_LOG=solana=trace _ cargo +$rust_nightly test --target-dir target/cov "${packages[@]}" 2> target/cov/coverage-stderr.log
+RUST_LOG=solana=trace _ cargo "$SOLANA_RUST_VERSION" test --target-dir target/cov --no-run "${packages[@]}"
```

Comment:
any better name? mabe just too generic? how about `SOLANA_RUST_VERSION_WITH_COVERAGE`?

Reply:
I don't think we need "SOLANA" in here.   What do you think about moving this override into `ci/rust-version.sh` instead:
```diff
-nightly_version=2019-11-13
+if [[ -n $RUST_NIGHTLY_VERSION ]]; then
+  nightly_version="$RUST_NIGHTLY_VERSION"
+else
+  nightly_version=2019-11-13
+fi
```
and same for stable_version

Reply:
Hm, I like this idea! a492f8eb74a1e398b6fc0668744c070487a98fb5